verilog/formatting/formatter_test.cc

```diff
@@ -2706,8 +2706,7 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
         "      if (cfg.enable) begin\n"
         "        count = 1;\n"
         "      end,\n"
-        "      utils_pkg::decrement()\n"
-        "  )\n"
+        "      utils_pkg::decrement())\n"
```

Comment:
If this was the only change to this test file needed, let's add some small test cases with just a macro call at the top-level, similar to function test cases with wrapped call arguments.

Reply:
Done